src/Stratis.Bitcoin.Features.Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -222,7 +225,7 @@ public Task SaveChangesAsync(IEnumerable<UnspentOutputs> unspentOutputs, IEnumer
                 {
                     transaction.ValuesLazyLoadingIsOn = false;
                     transaction.SynchronizeTables("BlockHash", "Coins", "Rewind");
-                    transaction.Technical_SetTable_OverwriteIsNotAllowed("Coins");
+                    //transaction.Technical_SetTable_OverwriteIsNotAllowed("Coins"); // Why it was there and what is it for? No one knows.
```

Comment:
Isn't this exactly to prevent the "coins" table from overwrites? I.e. overwriting existing data in the table? I would think that is an absolute must to include.

Reply:
but we want to override them (for example when some outputs were spend and some were not) and we did that before

Reply:
Why would we need it? I can rather imagine situations when this is the problem, which would be when you have transaction with multiple outputs which is in the database already and you just want to update one of the output. But maybe such an update is never done? But do we know that for sure? From the code it is not obvious for me why such an update could not happen.